front/package.json

```diff
@@ -20,7 +20,7 @@
   "dependencies": {
     "@auth0/nextjs-auth0": "^3.5.0",
     "@dust-tt/client": "file:../sdks/js",
-    "@dust-tt/sparkle": "^0.2.350",
+    "@dust-tt/sparkle": "^0.2.351-alpha",
```

Comment:
:x:

Reply:
Currently testing on front-edge, sparkle is not published yet, I will change once we publish it :)